sso_server/mail.py

```diff
@@ -0,0 +1,68 @@
+import smtplib
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+from jinja2 import Template
+
+from django.conf import settings
+
+
+class EmailSender:
+    def __init__(self, name, account=settings.EMAIL["ADDRESS"]):
+        self.smtp = None
+        self.account = account
+        self.name = name
+
+    def connect(self):
+        self.smtp = smtplib.SMTP_SSL(settings.EMAIL["HOST"], settings.EMAIL["PORT"])
+        self.smtp.ehlo()
+        pw = settings.EMAIL["PASSWORD"]
+        self.smtp.login(self.account, pw)
+
+    def sendemail(self, to, subject, text, cc=None, bcc=None, attachment=None, attachments=None, template=False, template_file_path=None, **kwargs):
+        msg = MIMEMultipart('alternative')
+        msg['Subject'] = "[Rezal] {}".format(subject)
+        msg['From'] = '{} <{}>'.format(self.name, self.account)
```

Comment:
Use f-strings, it's simpler:

```suggestion
        msg['Subject'] = f"[Rezal] {subject}"
        msg['From'] = f'{self.name} <{self.account}>'
```